tempest2/src/main/kotlin/app/cash/tempest2/Model.kt

```diff
@@ -204,6 +204,8 @@ class KeySet private constructor(
   inline fun <reified K : Any> getKeys(): List<K> {
     return getKeys(K::class)
   }
+
+  override fun toString(): String = contents.toString()
```

Comment:
originally was thinking of having assertions against the WriteSet and found these useful..ended up simplifying to the counters instead but may as well keep these?